subsys/bluetooth/controller/hal/nrf5/radio/radio.c

```diff
@@ -379,6 +379,11 @@ static void sw_switch(u8_t dir, u8_t phy_curr, u8_t flags_curr, u8_t phy_next,
 			HAL_SW_SWITCH_RADIO_ENABLE_PPI_REGISTER_TASK(ppi_en) =
 				HAL_SW_SWITCH_RADIO_ENABLE_PPI_TASK_TX;
 
+			/* Include PPI for S2 timing in the active group */
```

Comment:
The concept is: have only one PPI and only one CC for S2-switch, and add the PPI in the active group (i.e. for next Switch), if we are about to receive in LE Coded.